pom.xml

```diff
@@ -16,7 +16,7 @@
   <name>guacamole: variant caller</name>
 
   <properties>
-    <adam.version>0.18.1</adam.version>
+    <adam.version>0.19.0</adam.version>
     <bdg-formats.version>0.6.1</bdg-formats.version>
```

Comment:
Oh good call -- I wonder if that's related to my saveAsParquet issue

On Mon, Mar 21, 2016 at 11:41 AM, Arun Ahuja <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/hammerlab/guacamole/pull/396#discussion_r56843898>:
>
> > @@ -16,7 +16,7 @@
> >    <name>guacamole: variant caller</name>
> >
> >    <properties>
> > -    <adam.version>0.18.1</adam.version>
> > +    <adam.version>0.19.0</adam.version>
> >      <bdg-formats.version>0.6.1</bdg-formats.version>
>
> Also does this not require an update to bdg-formats version?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/guacamole/pull/396/files/e055a229727849da7831f900d49aa75bb6dea01c#r56843898>
>